dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard.md

```diff
@@ -0,0 +1,142 @@
+
+1. Create a table for merge and replicate data in the downstream database before the full data migration. Do not specify the `PRIMARY KEY` attribute for the `auto_pk_c1` column, but use column `auto_pk_c1` and `uuid_c2` to define the `PRIMARY KEY` instead.
+
+    ```sql
+    CREATE TABLE `tbl_multi_pk_c2` (
+      `auto_pk_c1` bigint(20) NOT NULL,
+      `uuid_c2` bigint(20) NOT NULL,
+      `content_c3` text,
+      PRIMARY KEY (`auto_pk_c1`,`uuid_c2`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1
+    ```
+
+2. Start the full and incremental data migration.
+
+3. Run `query-status` to verify that whether the data migration is successfully processed and whether the data from upstream have already been merged and replicated in the downstream database.
+
+## Create/drop tables in the upstream
+
+From [Merge and Replicate Data from Sharded Tables](/dev/reference/tools/data-migration/features/shard-merge.md#principles), it is clear that the coordination of sharding DDL lock depends on whether the downstream database receives the corresponding DDL statements of all upstream sharded tables. In addition, currently DM *does not* support dynamically creating or dropping sharded tables in the upstream. Therefore, to create or drop sharded tables in the upstream, it is recommended to perform the following steps.
+
+### Create sharded tables in the upstream
+
+If you need to create a new sharded table in the upstream, perform the following steps:
+
+1. Wait for the coordination of all executed sharding DDL in the upstream sharded tables to finish.
+
+2. Run `stop-task` to stop the task.
+
+3. Create a new sharded table in the upstream.
+
+4. Make sure that the configuration in `task.yaml` file allows the newly-added sharded table to be merged in one downstream table with other existing sharded table.
+
+5. Run `start-task` to start the task.
+
+6. Run `query-status` to verify that whether the data migration is successfully processed and whether the data from upstream have already been merged and replicated in the downstream database.
+
+### Drop sharded tables in the upstream
+
+If you need to drop a sharded table in the upstream, perform the following steps:
+
+1. Drop the existing sharded table, run [`SHOW BINLOG EVENTS`](https://dev.mysql.com/doc/refman/5.7/en/show-binlog-events.html) to fetch the corresponding `End_log_pos` in the binlog of the `DROP TABLE` statement, and mark it as *Pos-M*.
+
+2. Run `query-status` to fetch the position (`syncerBinlog`) corresponding to the binlog event that has been processed by DM, and mark it as *Pos-S*.
+
+3. When *Pos-S* is greater than *Pos-M*, it means that DM has finished processing the `DROP TABLE` statement, and the data of the table has been replicated to the downstream, so the subsequent operation can be performed. Otherwise, wait for DM to finish replicating the data.
```

Comment:
```suggestion
3. When *Pos-S* is greater than *Pos-M*, it means that DM has processed all of the `DROP TABLE` statements, and the data of the table before dropping has been replicated to the downstream, so the subsequent operation can be performed. Otherwise, wait for DM to finish replicating the data.
```